packages/icons/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/Icon.js",
   "scripts": {
     "prepare": "npm run clean && npm run build:svg && npm run build:index && npm run build:copy && npm run build:babel",
-    "clean": "rm -rf node_modules components lib",
+    "clean": "rm -rf components lib",
```

Comment:
?

Reply:
So `npm i` runs `prepare`.  If you remove the node_modules during `prepare` then it won't be able to run `svgr` or `babel` since there is no node_modules folder.  This is what I found while running node 8.15.0 and npm 6.4.1

Reply:
Happy to revert `package.json` if it's some sort of node or npm version issue, but curious if anyone else has seen these issues when working on pcln-icons

Reply:
Does `npm run prepare` run the `clean` npm script? The workflow I've been using is to run `npm run clean` to nuke all the `node_modules` and related dist folders, then run either `npm i` (with runs postinstall script to hook into `prepare` script, or run `npm run prepare`)

I think it'd be safer to revert this package.json change for pcln-icons to move forward with this PR, and have a follow-up convo / PR to adjust any nuance about pcln-icons build / publishing workflow.

Reply:
Does `npm run prepare` run the `clean` npm script? The workflow I've been using is to run `npm run clean` to nuke all the `node_modules` and related dist folders, then run either `npm i` (with runs postinstall script to hook into `prepare` script, or run `npm run prepare`)

I think it'd be safer to revert this package.json change for pcln-icons to move forward with this PR, and have a follow-up convo / PR to adjust any nuance about pcln-icons build / publishing workflow.

Reply:
i think the disconnect is that the tests are meant to be run from the root of the monorepo, not from inside the package

Reply:
I see. Yes, I had been running the tests from the icons folder

Reply:
UPDATED . Thanks folks!